sdk/typescript/src/graphql/client.ts

```diff
@@ -64,7 +64,7 @@ export class SuiGraphQLClient<Queries extends Record<string, GraphQLDocument>> {
 		this.#url = url;
 		this.#queries = queries;
 		this.#headers = headers;
-		this.#fetch = fetchFn;
+		this.#fetch = (...args) => fetchFn(...args);
```

Comment:
this changes the "this" binding of fetch.  `window.fetch()` and `fetch = window.fetch; fetch()` are both valid, but obj = { fetch: window.fetch }; obj.fetch()` will error